lstm_rnn.py

```diff
@@ -233,11 +261,15 @@ def on_epoch_end(epoch, logs):
 
     # SET THE TRAINING PARAMETERS, THEN FIT THE MODEL
     # TODO - EXPERIMENT: try training with different # of batch sizes and epochs
-    filename = "generated_text_" + str(time.time())
-    examples_file = open("./gentext/"+filename, "w")
-    model.fit_generator(generator(sentences, next_words, BATCH_SIZE),
-                        steps_per_epoch=int(len(sentences)/BATCH_SIZE) + 1,
-                        epochs=100,
-                        callbacks=callbacks_list,
-                        validation_data=generator(sentences_test, next_words_test, BATCH_SIZE),
-                        validation_steps=int(len(sentences_test)/BATCH_SIZE) + 1)
+    gen_filename = "gen_text_" + re.sub('\.txt$', '', filename) + "_" + datetime.datetime.now().strftime("%y-%m-%d-%H-%M")
+    examples_file = open("./gentext/"+gen_filename, "w")
```

Comment:
Should we close the file before exiting the program?
```
examples_file.close()
```
i.e. put this on line 275 below.